docs/ramp_fitting/source/main.rst

```diff
@@ -12,7 +12,7 @@ data cube in a second output data product.  A third, optional output product
 is also available and is produced only when the step parameter 'save_opt' is
 True (the default is False).  The output values will be in units of counts (DN)
```

Comment:
Also, further up at line 8 there's a reference to the primary output file being "RampFit". I would change that to "rate" or "_rate". And then mention that the output file for multiple integrations is "rateints".